utils/build-script-impl

```diff
@@ -1824,6 +1824,16 @@ function set_swiftsyntax_build_command() {
         SWIFT_TEST="$(build_directory_bin ${LOCAL_HOST} swiftpm)/swift-test"
     fi
 
+    if [ "${BUILD_LIBPARSER_ONLY}" ]; then
+        # we don't have a compiler built so we have to use the one in the environment.
+        SWIFTC_USE="$(xcrun_find_tool swiftc)"
```

Comment:
Nitpick but `SWIFTC_BIN` is better name here, it's used in other places in this script representing the same thing as here.

Reply:
Done